src/senaite/core/browser/form/adapters/sampletype.py

```diff
@@ -19,13 +19,19 @@
 # Some rights reserved, see README and LICENSE.
 
 from senaite.core.browser.form.adapters import EditFormAdapterBase
+from senaite.core.interfaces import ISampleType
 from bika.lims.vocabularies import getStickerTemplates
 
+_DGF_WIDGET_PREFIX = "form.widgets.admitted_sticker_templates.0.widgets."
+
 
 class EditForm(EditFormAdapterBase):
     """Edit form adapter for Sample Type
     """
 
+    def __init__(self, context, request):
+        super(EditForm, self).__init__(context, request)
+
```

Comment:
You can remove this `__init__`. Since you don't do anything else here other calling the base class default, there is no need to overwrite the constructor.